### DISCUSSION TITLE
Guidelines for handling docs with more levels than furo supports

### DISCUSSION THREAD

**Discussion by astafan8** [pradyunsg/furo#491] [Questions] [answered]
Dear all, especially furo devs and contributors, I learned about this theme recently, and it's indeed great, swift, clear, and seems to be becoming a standard. So I started working on using it for docs on a project I work on, and I am facing an issue with rendering, see below.

This is with read-the-docs theme that is used at the moment:
![image](https://user-images.githubusercontent.com/15662810/181751894-67a50be8-5376-4711-8c00-b48bb578fe94.png)

and this is with furo:
![image](https://user-images.githubusercontent.com/15662810/181751526-9b28d410-01bf-4620-a9ec-0534e8566381.png)

the issue is that furo flattens levels of the documentation. I read in the furo documentation that the theme is meant for 1-2 levels of docs and more is not really supported. So, I'd love some help and advice with how to proceed:
- is there a setting in furo that i can adjust / play with to make the rendering show more levels?
- is it recommeded to restructure the docs to have less levels?
- any other suggestions?

Looking forward to your advice here!

**Comment by pradyunsg** [comment, marked as answer]
Those look like they were headings within the content, which show up on the right sidebar with Furo. Separating the page content hierarchy and site structure was an explicit design goal for Furo.

Conceptually, the left sidebar is the site structure (as defined by the `toctree` directives), providing site-wide navigation and visibility. The right sidebar has the headings "on the page", which provide in-page navigation.
